src/app/p-common/menubar/menubar.component.ts

```diff
@@ -9,17 +9,22 @@ import { MenuItem } from 'primeng/api';
 export class MenubarComponent implements OnInit {
 
   items: MenuItem[];
+  currentProject: string;
 
   constructor() {
+    this.currentProject = localStorage.getItem('papaya-project');
```

Comment:
to raczej nie może być w localStorage, trzeba zrobić jakiś serwis na takie rzeczy

Reply:
okej, czy mogę rozszerzyć `projectService` o metodę `chooseProject`? czy oddzielny serwis być to musi?

Reply:
hmm, no na razie możesz 
w innym zadaniu też potrzebuję w stanie aplikacji trzymać wybrany projekt, więc zrobiłem pod to dedykowany serwis, ale żeby się nie konflikciło, to zrób to tak

Reply:
zanim odpowiedziałeś, to zrobiłem oddzielny `projectService`, który trzyma `currentProject`. 
pozostałe uwagi również poprawiłem, sprawdź proszę.

Reply:
wait, jeszcze chwila, coś się wykopyrtło.

Reply:
w sumie to nie zmieniłeś żeby korzystało ze storea xd